core/src/test/java/org/arquillian/smart/testing/configuration/ObjectMapperTest.java

```diff
@@ -7,11 +7,14 @@
 import java.util.List;
 import java.util.Map;
 import java.util.function.Function;
+import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 import org.junit.After;
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.contrib.java.lang.system.RestoreSystemProperties;
```

Comment:
still unnecessary imports?

Reply:
Yeah, I was thinking about the same - maybe style checker could help

Reply:
This import is necessary for now, but yeah, even walkmod could fix this automatically.